packages/components/switch/src/Switch.tsx

```diff
@@ -6,12 +8,45 @@ import { Label } from './SwitchLabel'
 export type SwitchProps = InputProps
 
 export const Switch = forwardRef<HTMLButtonElement, SwitchProps>(
-  ({ value = 'on', size = 'md', children, className, ...rest }, ref) => {
+  ({ value = 'on', size = 'md', children, className, id, disabled, ...rest }, ref) => {
+    const { id: controlledInputId, labelId: controlledLabelId } = useFormFieldState()
+
+    const switchId = useId(id)
```

Comment:
IMO `switch` prefix is not needed it's redundant

Reply:
I understand your point. Wdyt of `innerId`, for example?